Formula/clang-format.rb

```diff
@@ -4,10 +4,10 @@ class ClangFormat < Formula
   version "2016-12-09"
 
   stable do
-    url "http://llvm.org/svn/llvm-project/llvm/tags/google/stable/2016-12-09/", :using => :svn
+    url "https://llvm.org/svn/llvm-project/llvm/tags/google/stable/2016-12-09/", :using => :svn
```

Comment:
Apparently not, since it failed to fetch on test-bot. @MikeMcQuaid this is a bug in the new audit code.